src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/AddUserFormSubscriber.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Bundle\UserBundle\Form\EventSubscriber;
+namespace Sylius\Bundle\CoreBundle\Form\EventSubscriber;
```

Comment:
CustomerType use it, when we create Customer in Sylius admin, if I am not mistaken.